IronSoftware.Drawing/IronSoftware.Drawing.Common/AnyBitmap.cs

```diff
@@ -2182,102 +2550,88 @@ protected virtual void Dispose(bool disposing)
                     return;
                 }
 
-                Image?.Dispose();
-                Image = null;
+                foreach (var x in GetInternalImages() ?? [])
```

Comment:
This will throws an exception when corrupt or invalid image. For `Dispose` should not throw an exception.

To test this you can use:
```csharp
// Arrange
byte[] invalidMagicBytes = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
var anyBitmap = new AnyBitmap(invalidMagicBytes);

// Act
var exception = Record.Exception(() => anyBitmap.Dispose());

// Assert
Assert.Null(exception);
```

Reply:
As I tested it load fine when `var imageBitmap = AnyBitmap.FromBytes(bytes, false);` but it throws the exception when `Dispose`. But it occur only .NET 472 which means it used older version of `SixLabor.ImageSharp`.